docs/docs/stylesheets/ci.css

```diff
@@ -1,48 +1,12 @@
-.md-header {
-    background-color: #606A71;
+:root {
```

Comment:
guess we can drop the whole custom css file and rely just on the `slate` color palette from mkdocs to avoid any css issues in the future.

Reply:
Hi Carsten,
we can drop the custom css, but then the design is even further away from flamingo.me, and I was tasked to update the design to match flamingo.me (the dark pages) more including CSS modifications. This is also true for the changes below, as theme "slate" and primary color "white" do not work well on this theme.
I will redirect your request to my original stakeholder Daniel P so that you two can decide how to proceed.